src/Compilers/Core/VBCSCompiler/BuildProtocol.cs

```diff
@@ -259,8 +258,8 @@ public void WriteToBinaryWriter(BinaryWriter writer)
     ///
     /// Field Name       Field Type          Size (bytes)
     /// -------------------------------------------------
-    /// responseLength   int (positive)      4  
     /// responseType     enum ResponseType   4
+    /// responseLength   int (positive)      4  
```

Comment:
I don't think this change is correct. i.e. I think this was OK before.
The length is written first, then the "payload" from the BinaryWriter which leads with the 'type'